fares-data-build-tool/create-data
# feat: Added requires attention logic to pointToPointProducts page

# Description

-   Added requires attention logic to pointToPointProducts page.

# Testing instructions

-   Run locally and create a product, and then manually set the value to true in the code, to see the change on the UI.

# Type of change
